portfolio/src/main/webapp/script.js

```diff
@@ -27,27 +27,37 @@
 //   const greetingContainer = document.getElementById('greeting-container');
 //   greetingContainer.innerText = greeting;
 // }
-function loadTasks() {
+
+// function loadcomment() {
+//     fetch('/form-content').then(res => loadTasks());
+// }
+
+function loadComments() {
   fetch('/form-content').then(response => response.json()).then((tasks) => {
-    const comment = document.getElementById('data-container');
-    tasks.forEach((task) => {
-      comment.innerText=task;
+    const comment = document.getElementById('comment-container');
+    tasks.forEach((newMessage) => {
+       comment.appendChild(createComment(newMessage));
     })
   });
 }
-function createParagraph(text) {
-  const paragraph = document.createElement('p');
-//   body.querySelector('#image-Url').src = listing.imageUrl;
-  paragraph.innerText = text;
-  return paragraph;
+function createComment(newMessage) {
```

Comment:
Check out how you folks used templates in COVIDConnect. It enables complex HTML in each comment representation without creating the structure from javascript.